pubspec.yaml

```diff
@@ -17,3 +17,4 @@ dev_dependencies:
   lints: ^1.0.1
   path: ^1.8.0
   test: ^1.17.12
+  coverage:
```

Comment:
package:test has a dependency on `coverage`, so I don't think we need a constraint.